src/plugins/discover/public/application/main/data_fetching/fetch_esql.ts

```diff
@@ -40,14 +41,15 @@ export function fetchEsql({
   query: Query | AggregateQuery;
   inputQuery?: Query;
   filters?: Filter[];
+  inputTimeRange?: TimeRange;
```

Comment:
This makes it so that ES|QL saved searches now respond to the dashboard timeslider - previously, the timeslice was ignored.